starfish/test/image/test_slicedimage_dtype.py

```diff
@@ -0,0 +1,116 @@
+import warnings
+
+import numpy
+import pytest
+from slicedimage import Tile, TileSet
+
+from starfish.constants import Coordinates, Indices
+from starfish.errors import DataFormatWarning
+from starfish.image import ImageStack
+
+
+NUM_HYB = 2
+NUM_CH = 2
+NUM_Z = 2
+HEIGHT = 10
+WIDTH = 10
+
+
+def make_stack(dtype: numpy.number, corner_dtype: numpy.number):
```

Comment:
Could this be combined with the [synthetic_stack](https://github.com/spacetx/starfish/blob/master/starfish/test/dataset_fixtures.py#L31) fixture? 

If not, it's probably still a good idea to make it a fixture and move it to that file so that other tests could load this and test against it. 

N.B.: If confusing, can see how the fixtures work by checking `test_apply.py`, which consumes the `synthetic_stack` tile function you'd written previously and that I converted to a fixture.

Reply:
#268